mcs/class/System.Core/Makefile

```diff
@@ -31,11 +31,17 @@ ifneq (2.1, $(FRAMEWORK_VERSION))
 LIB_MCS_FLAGS += -d:NET_3_5
 endif
 
-ifdef AOT_FRIENDLY_PROFILE
-extra_test_flags := -exclude:NotWorkingLinqInterpreter
-LIB_MCS_FLAGS += -d:NO_FEATURE_STATIC_DELEGATE,FEATURE_MAKE_RUN_METHODS
+MOBILE_NATIVE_INTERP :=
+
+ifdef MOBILE_NATIVE_INTERP
```

Comment:
how is this supposed to be used?

Reply:
We enable it for profiles which supports SRE interpreter (e.g. wasm, monotouch)